### DISCUSSION TITLE
Tier 3 items dropped in AQ40

### DISCUSSION THREAD

**Discussion by NoodlestheNoble** [chromiecraft/chromiecraft#3903] [Suggestions]
So this is going to seem a little out there but please hear me out, I hear a lot of people wanting T3 set for Tmog from Naxx, a raid we wont have here (Which I am happy for honestly) and as the simpleton I am I realised there are 9 pieces in the set and 9 bosses in AQ40. What's the chances on the possibility to add them to the boss loot? I have zero clue about scripting or how easy/difficult it would be to implement this. I do know it would probably become harder if the stats on the gear needed to be reduced too but again I don't know if that would also be necessary.

I think it would help to satisfy the people that want to get T3 without the need to even consider adding Naxx 60 into the mix, which I'm like 99% sure if more of a scripting nightmare than 9 items!

**Comment by Frodhi** [comment]
Adding the items to the loot table is not hard.
What is kinda tricky would be the fact that the t3 gear is superior to the AQ one.
So i would kinda mess up the progression.
Maybe if there was an option of a slightly more tuned AQ that drops t3?

**Reply by NoodlestheNoble** [reply]
Something like that or the stats on the items could be lowered. I feel like that would be slightly harder to do than just adding them as they are supposed to be but long term its the last raid in classic, so as long as it doesn't really effect the TBC progression in a way that would hinder the natural progress there I don't personally see too big of an issue in some pieces of AQ gear get a little scuffed along the way.
Of course that's personal opinion and I know not everyone would like that.

**Comment by Cooties-cc** [comment]
A very simple fix to this problem would be to change the value of the gear to zero stats. It would just become a piece of vanity gear, which you could equip so it can be added to your transmog list. Similar to things they did later in the game, where they added pieces of old unobtainable gear that people could use as transmog. It wouldn't hurt anything gameplay wise, as they would not have stats or set bonuses, or anything on them, however the difficulty in obtaining them (aq40 vs 60naxx) would be "easier". It could also give more reasons to continue to raid aq40 even when you're in BiS fully geared. 
![image](https://user-images.githubusercontent.com/62584589/184406091-7cb5546e-cc0e-4da7-89b0-21f04275e59f.png)
The halloween helm is a perfect example at a piece of transmog only gear. It has no stats (mostly), is rarity green+, and has a armor type. 
T3 could be changed into something like this. 
**should also include that these would vendor for 1 copper, and could not be disenchanted. This prevents extra materials being given to a raid that otherwise would not occur.

**Reply by NoodlestheNoble** [reply]
That is probably the best idea tbh. Gives everyone what they wanted without being overpowered

**Reply by balleny** [reply]
but adding additional (new) items, would require users to install an additional patch file, which is drifting far away from CCs main focus.

**Reply by Cooties-cc** [reply]
There is no new additional items. Wotlk simply made the old items unobtainable, as they weren't destroyed upon installing the new expansion. This is why those who still had them made them that much more valuable. You can say the same about Onyxia at level 60, which was removed with the wotlk patch 3.2.2

![image](https://user-images.githubusercontent.com/62584589/184504972-423082d4-1446-49fe-b831-04d30fd77f2c.png)
We still get level 60 Onyxia loot without having to download and install new MPQ files, correct? So adding the loot tables from naxx60, with their stats/bonuses removed, to the existing AQ40 bosses wouldn't be an issue.

**Reply by balleny** [reply]
yep, and if you want vanity items without stats, you would need to add new items.
